common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
I would change colors to increase the contrast.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122671827-8be3c280-d1d9-11eb-8f15-96305b9dd117.png)

</details>